### PR TITLE
Add cloudwatch alarm policy

### PR DESCRIPTION
**Pull request by gianargamosa**
Cloudwatch alarms policy.

### CODE REVIEW

File: policies/setting-up-alarms.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+For each metric, configure alarms with the following settings:
+
+- **Evaluation Period**: Minimum of **5 minutes**.
+- **Datapoints to Alarm**: Use **2 out of 3** evaluation periods to reduce false positives.
+- **Actions**: Send alerts and, where possible, automate responses.
+
+### Notification Channels
+Set up notification channels to ensure alerts reach the right people:
+
+1. **Slack**: Use Slack **cloudwatch-alarm-notification** channel for critical alerts, such as when Lambda functions fail or X-Ray faults spike.
```
Comment (mbdelarosa) [inline policies/setting-up-alarms.md]: Ohh iba pa ba yung channel na ito sa `prod-sqs-lambda-alerts` channel natin ngayon?

File: policies/setting-up-alarms.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+---
+
+## 6. Policy Compliance
+
+- All alarms and thresholds must be documented and reviewed by the DevOps team.
+- Any changes to alarm settings must be approved and updated in the documentation.
+
+---
+
+This policy ensures that booky services using AWS Lambda and X-Ray are continuously monitored, allowing us to address issues before they affect the user experience.
```
Comment (mbdelarosa) [inline policies/setting-up-alarms.md]: Pa-capitalize nalang si Booky here haha

File: policies/setting-up-alarms.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+### Scope
+This applies to all developers, DevOps engineers, and technical personnel responsible for managing and monitoring the booky services deployed on AWS Lambda.
+
+---
+
+## 1. Metrics to Monitor
+
+We’ll monitor key Lambda and X-Ray metrics to ensure everything is running smoothly:
+
+1. **Lambda Function Error Rate**
```
Comment (mbdelarosa) [inline policies/setting-up-alarms.md]: Mas okay bang may colon nalang din after the title dito? Para lang ma-separate kasi nare-render sila in one line kasama nung text sa ibaba 😅 

image

Same with lines 20 and 25

File: policies/setting-up-alarms.md (hunk 1 above)
Comment (gianargamosa) [reply policies/setting-up-alarms.md]: Ahh yan sana kasi yung gagawin ko na channel kaso pinabago tapos di ko na na-adjust dito, will change po to the specific channel @mbdelarosa

**Review by mbdelarosa — APPROVED** [review]

**Review by alecguintu — APPROVED** [review]

**Pull request merged**